generators/csharp/csharp.sln

```diff
@@ -0,0 +1,73 @@
+﻿
```

Comment:
is this intentional?

Reply:
No, I don't know why this is added.

Reply:
Gotcha! Just make sure to remove before merging :)